Formula/ecl.rb

```diff
@@ -14,19 +14,39 @@ class Ecl < Formula
   end
 
   depends_on "gmp"
+  depends_on "bdw-gc"
 
   def install
-    system "./configure", "--prefix=#{prefix}",
+    # These environment variables are necessary or gmp / boehmgc won't be found
+    # No idea why
+    gmp = Formula["gmp"]
+    boehmgc = Formula["boehmgc"]
+    ENV.append "CFLAGS", "-I#{gmp.opt_include} -I#{boehmgc.opt_include}"
+    ENV.append "CXXFLAGS", "-I#{gmp.opt_include} -I#{boehmgc.opt_include}"
+    ENV.append "LDFLAGS", "-L#{gmp.opt_lib} -L#{boehmgc.opt_lib}"
+
+    # https://gitlab.com/embeddable-common-lisp/ecl/tree/revert-make-split
+    ENV.deparallelize
+
+    system "./configure",
+                          "--prefix=#{prefix}",
                           "--enable-threads=yes",
-                          "--with-system-gmp=yes"
+                          "--with-dffi=included",
+                          "--with-cxx",
+                          "--enable-gmp=system"
     system "make"
     system "make", "install"
+    system "make", "check"
```

Comment:
> @gitfoxi Ping on this. Would be good to file an upstream issue to ask about this.

For some reason, there's no reply box for the above. One issue is that upstream wants to deprecate use of system installed libraries. They want to use sources for BDWGC and GMP that they include in ECL's own source package. I don't agree with their approach, not least because I can't make it work on homebrew, but more because ECL is supposed to be embeddable as a library in other code bases. What if that other, host, code happens to use BDWGC or GMP? What if it happens to require different versions? Things could get (and have gotten) ugly.